akka-js-actor/js/src/main/scala/akka/dispatch/ExecutorServiceDelegate.scala

```diff
@@ -11,7 +11,7 @@ class EventLoopExecutor extends ExecutorServiceDelegate {
   def executor: ExecutorService = this
 
   final val deferringFunction: Runnable => Unit =
-    if (js.isUndefined(global.setImmediate)) {
+    if (js.typeOf(global.setImmediate) == "undefined") {
```

Comment:
A better check is `js.typeOf(global.setImmediate) == "function"` since you can pollute the global namespace with anything. Doesn't buy you much, but it is a little more idiomatic for JavaScript/TypeScript imo.

Reply:
make sense! This week is scheduled a new Akka release, I'm going to batch the fix and the upgrade there.